### DISCUSSION TITLE
Request for Improvement: Enable TestOps plugin for JiraDC for Issue with selected types

### DISCUSSION THREAD

**Discussion by NekrasovAM** [allure-framework/allure2#2323] [TestOps]
Hi! 
Can u make that the Allure TestOps plugin for JiraDC can be enabled only for issues with the selected types, please? This would be very useful for all!
I receive negative feedback from colleagues that they are bothered by the constantly opened "**Allure: Test Cases**" and "**Allure: Launches**" blocks in issues with Epic, Story and other types not related to testing.
It would be convenient to add an additional option in the plugin settings window, which is in the screenshot.
![testops_for_jira](https://github.com/allure-framework/allure2/assets/3220426/b2cd54d2-20c2-4dbd-a1c1-1daacd3445b4)

**Comment by cheshi-mantu** [comment]
Hello @NekrasovAM 
could you please file a Feature request ticket at https://help.qameta.io?
We support Allure Testops and its ecosystem via dedicated support portal and team.
If you have any ideas how else we could enhance the plugin, please do not hesitate to add your other suggestions.
thanks in advance

**Comment by aikunim-yes** [comment]
Hello! we need same configurations for Jira Cloud.
